Content.Shared/Humanoid/SharedHumanoidAppearanceSystem.cs

```diff
@@ -250,6 +250,26 @@ public void SetSex(EntityUid uid, Sex sex, bool sync = true, HumanoidAppearanceC
         }
     }
 
+    // Parkstation-HeightSlider Start
+    /// <summary>
+    ///     Set a humanoid mob's sex. This will not change their gender.
```

Comment:
fucking what

Reply:
can't stop laughing at this, I knew I needed to change the comment and somehow only did the parameter